tests/functionaltests/FunctionalTests.cpp

```diff
@@ -691,7 +697,7 @@ class ProjectionTest {
     }
 
     TEST_METHOD_CLEANUP(ProjectionTestCleanup) {
-        FunctionalTestCleanupUIApplication();
+        (void)FrameworkHelper::RunOnUIThread(&FunctionalTestCleanupUIApplication);
```

Comment:
in all places

---
In reply to: [94668735](https://github.com/Microsoft/WinObjC/pull/1627#discussion_r94668735) [](ancestors = 94668735)